src/app/api/routes/media.py

```diff
@@ -90,7 +92,9 @@ async def upload_media(media_id: int = Path(..., gt=0),
     Upload a media (image or video) linked to an existing media object in the DB
     """
     entry = await check_for_media_existence(media_id, current_device.id)
-    bucket_key = str(hash(datetime.utcnow()))
+
+    # Concatenate the first 32 chars of SHA256 hash with file extension
```

Comment:
Maybe mention that 32 is a bit of  a "random" threshold but it should be enough and it will prevent us from meeting issues with some systems interactions.

Reply:
done!